staging/src/k8s.io/api/batch/v1/generated.proto

```diff
@@ -184,3 +267,29 @@ message JobStatus {
   optional int32 failed = 6;
 }
 
+// JobTemplate describes a template for creating copies of a predefined pod.
+message JobTemplate {
+  // Standard object's metadata.
+  // More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  // +optional
+  optional k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
+
+  // Defines jobs that will be created from this template.
+  // https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  // +optional
+  optional JobTemplateSpec template = 2;
```

Comment:
This doesn't look like a optional field. We have strict validation against the contents.

Reply:
see above